packages/fast-components/src/styles/patterns/button.ts

```diff
@@ -0,0 +1,199 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../display";
+import { focusVisible } from "../focus";
+import { heightNumber } from "../size";
+
+export const BaseButtonStyles = css`
+    ${display("inline-block")} :host {
+        font-family: var(--body-font);
+        outline: none;
+    }
+
+    .control {
+        ${
+            /* Font size is temporary - 
+            replace when adaptive typography is figured out */ ""
+        } font-size: 14px;
+        line-height: 20px;
+        line-height: 1;
+        box-sizing: border-box;
+        display: inline-flex;
+        justify-content: center;
+        align-items: center;
+        padding: 0 calc((6 + (var(--design-unit) * 2 * var(--density))) * 1px);
+        height: calc(${heightNumber} * 1px);
+        min-width: calc(${heightNumber} * 1px);
+        max-width: calc(var(--design-unit) * 94.5px));
```

Comment:
I think we should remove the max width. It doesn't add anything and it's an arbitrary value. Basic guidance should say don't make really long buttons, but I don't think there's value in enforcing it.

Reply:
I'm for this